eng/pipelines/templates/steps/analyze.yml

```diff
@@ -10,8 +10,15 @@ steps:
 
   - script: |
      pip install -r eng/ci_tools.txt
-     ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/.docsettings.yml
-    displayName: 'Verify Readmes'
+     ward scan -d $(Build.SourcesDirectory) -c $(Build.SourcesDirectory)/.docsettings.yml -t all
```

Comment:
Do we want to do this now or after we enable the changelog update as part of version increment? I guess for python they already validate the changelog so presumably this wouldn't add any extra burden, but we should switch off the other validation now that we are using doc-warden.